src/odemis/driver/smartpod.py

```diff
@@ -119,38 +140,40 @@ class SmartPodDLL(CDLL):
     all the functions automatically.
     """
     
+    hwModel = c_long(10001)  # specifies the SmarPod 110.45 S (nano)
```

Comment:
Let's name that constant based on what it's for -> SMARPOD_110_45_S.
You also don't need to put in the DLL class. If you put it at the top of the file, it's easier to write afterwards.